src/main/java/uk/gov/companieshouse/certifiedcopies/orders/api/service/CertifiedCopyItemService.java

```diff
@@ -34,12 +43,21 @@ public CertifiedCopyItem createCertifiedCopyItem(final CertifiedCopyItem certifi
         certifiedCopyItem.setEtag(etagGenerator.generateEtag());
         certifiedCopyItem.setLinks(linksGenerator.generateLinks(certifiedCopyItem.getId()));
 
+        String description = descriptionProvider.getDescription(certifiedCopyItem.getData().getCompanyNumber());
+        certifiedCopyItem.setDescriptionIdentifier(DESCRIPTION_IDENTIFIER);
+        certifiedCopyItem.setDescription(description);
+        certifiedCopyItem.setKind(KIND);
+
+        Map<String, String> descriptionValues = new HashMap<>();
+        descriptionValues.put(DESCRIPTION_IDENTIFIER, description);
+        descriptionValues.put(COMPANY_NUMBER_KEY, certifiedCopyItem.getData().getCompanyNumber());
+        certifiedCopyItem.setDescriptionValues(descriptionValues);
```

Comment:
Block could be a single method call for clarity?

Reply:
So one method to set the description and one for the description values? Or just one method called setDescriptions which sets the description and the description values?

Reply:
Probably not that important. I note you formatted the code in two blocks, but if they're really part of the same action, a single method would do.